### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by shivampandey0**
Implemented the Avatar component

- Following the folder structure from [Bulma](https://github.com/jgthms/bulma/tree/master/sass)
- The `index.html` in  `css`  folder is for development purposes only and to visualize the components while building.
- The `helpers.css` contains a few classes for containers I needed for the `index.html`, I wasn't sure where to put them so right now they are acting as helpers so that is why I've put them here.

### CODE REVIEW

File: css/base/base.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@import "reset.css";
+
+/* global variables */
+:root{
+    --primary-color: #E40C0B;
```
Comment (ttahm3d) [inline css/base/base.css]: Try to format the code using prettier.

File: css/components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    justify-content: center;
+    align-items: center;
+    background-color: var(--primary-color);
+    color: var(--text-white);
+}
+
+.avatar-lg{
+    width: 8rem;
+    height: 8rem;
+    font-size: 2rem;
```
Comment (lannisterrr) [inline css/components/avatar.css]: font-sizes can go to a separate typography's style guide .

File: css/base/base.css (hunk 1 above)
Comment (coldpigli) [inline css/base/base.css]: Just a **suggestion**: Try to use HSL format for colors instead of Hex-codes.

Comment (shivampandey0) [reply css/base/base.css]: Hey Tahir, done the formatting please take a look.

File: css/components/avatar.css (hunk 2 above)
Comment (shivampandey0) [reply css/components/avatar.css]: right now I'm doing the components independently in the end I might do this but for now, keeping the things simple.

File: css/base/base.css (hunk 1 above)
Comment (shivampandey0) [reply css/base/base.css]: hey, I've updated the colors with HSL format, thanks for the suggestion.

**Pull request merged**